cli/tests/pcluster/config/utils.py

```diff
@@ -389,6 +393,11 @@ def init_pcluster_config_from_configparser(config_parser, validate=True, auto_re
         with open(config_file.name, "w") as cf:
             config_parser.write(cf)
 
+        if "AWS_DEFAULT_REGION" not in os.environ:
+            # We need to provide a region to PclusterConfig to avoid no region exception.
+            # Which region to provide is arbitrary.
+            os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
```

Comment:
Instead of mocking the `init_pcluster_config_from_configparser` I'd mock the tests using it, by adding the region in the configuration file or in the dictionary parsed to create the PclusterConfig object

In this way the region will came from the test and not from this initializator.